dashboard/app/models/ability.rb

```diff
@@ -191,8 +191,12 @@ def initialize(user)
       end
     end
 
+    can [:vocab, :resources, :code, :standards], UnitGroup do
+      true
+    end
+
     # Override Script and ScriptLevel.
-    can :read, Script do |script|
+    can [:read, :vocab, :resources, :code, :standards], Script do |script|
```

Comment:
I would suggest splitting these additions into their own block, so that you can add a clause for `is_migrated`